recipe/meta.yaml

```
@@ -99,189 +87,92 @@ outputs:
         - {{ cdt('pciutils-devel') }}        # [linux]
         - {{ cdt('libxscrnsaver-devel') }}   # [linux]
         - {{ cdt('libxtst-devel') }}         # [linux]
-        - {{ cdt('libselinux') }}            # [linux]
         - {{ cdt('libselinux-devel') }}      # [linux]
-        - {{ cdt('libxdamage') }}            # [linux]
         - {{ cdt('libxdamage-devel') }}      # [linux]
-        - {{ cdt('libxfixes') }}             # [linux]
         - {{ cdt('libxfixes-devel') }}       # [linux]
-        - {{ cdt('libxxf86vm') }}            # [linux]
         - {{ cdt('libxxf86vm-devel') }}      # [linux]
         - {{ cdt('libsm-devel') }}           # [linux]
         - {{ cdt('libice-devel') }}          # [linux]
         - {{ cdt('libxcb') }}                # [linux]
-        - {{ cdt('libxcb-devel') }}          # [linux and not (aarch64 or x86_64)]
-        - {{ cdt('expat-devel') }}           # [linux and not aarch64]
-        - {{ cdt('pcre') }}                  # [linux and not x86_64]
-        - {{ cdt('libglvnd-glx') }}          # [linux and not x86_64]
-        - {{ cdt('libglvnd-egl') }}          # [linux and not x86_64]
-        - {{ cdt('libxkbcommon-devel') }}    # [linux and not (aarch64 or x86_64)]
-        - {{ cdt('xcb-util-devel') }}        # [linux and not (aarch64 or x86_64)]
-        - {{ cdt('xcb-util-image-devel') }}  # [linux and not (aarch64 or x86_64)]
-        - {{ cdt('xcb-util-keysyms-devel') }}     # [linux and not (aarch64 or x86_64)]
-        - {{ cdt('xcb-util-renderutil-devel') }}  # [linux and not (aarch64 or x86_64)]
-        - {{ cdt('xcb-util-wm-devel') }}          # [linux and not (aarch64 or x86_64)]
+        - {{ cdt('expat-devel') }}           # [linux and x86_64]
+        - {{ cdt('pcre') }}                  # [linux and aarch64]
+        - {{ cdt('libglvnd-devel') }}        # [linux and aarch64]
         - jom                                # [win]
-        - make  # [not win]
+        - make                               # [not win]
         - pkg-config
       host:
         - python
-        - pyqt-builder >=1.14.1,<2
-        - sip >=6.6.2,<7
-        - qt-main 5.15.*
-        - {{ cdt('mesa-libgl-devel') }}          # [linux]
-        - {{ cdt('mesa-libegl-devel') }}         # [linux]
+        - pyqt-builder >=1.15,<2
+        - sip >=6.8,<7
```

```suggestion
        - sip >=6.8.6,<7
```
from pyproject.toml:
`requires = ["sip >=6.8.6, <7", "PyQt-builder >=1.15, <2"]`

The same applies for the next output PyQt6